examples/adc.js

```diff
@@ -43,27 +42,29 @@ async function main() {
 
 /**
  * Instead of specifying the type of client you'd like to use (JWT, OAuth2, etc)
- * this library will automatically choose the right client based on the environment.
+ * this library will automatically choose the right client based on the
+ * environment.
  */
-async function getADC() {
-  // Acquire a client and the projectId based on the environment. This method looks
-  // for the GCLOUD_PROJECT and GOOGLE_APPLICATION_CREDENTIALS environment variables.
+async function
+getADC() {
```

Comment:
It looks quite weird to me to split `async function` and the function name across two lines, but if this is what the tool does, ok.